spec/app_spec.rb

```diff
@@ -248,14 +248,14 @@
         expect(CGI.unescape(last_response.location)).to include('vtr=["C1"]')
       end
 
-      xit 'redirects to an ialmax sign in link if ial param is 0' do
+      it 'redirects to an ialmax sign in link if ial param is 0' do
         get '/auth/request?ial=0'
 
         expect(last_response).to be_redirect
         expect(last_response.location).to include(
           'scope=openid+email+social_security_number',
         )
-        expect(CGI.unescape(last_response.location)).to include('vtr=["C1.P1"]')
+        expect(CGI.unescape(last_response.location)).to include('vtr=["C1.P1","C1"]')
```

Comment:
Should this be `["C1","C1.P1"]` since the first vector is selected if the user is not identity verified?

Reply:
This should be `P1.C1, C1`. The authn context resolver will look for a vector that does not require proofing and use one if it exists.